packages/state-transition/src/stateTransition.ts

```diff
@@ -187,11 +189,24 @@ export function processSlots(
  *   ╚═══════════════════════════════════════════════════════════════════════════════════╝
  *   beacon-node ShufflingCache
  */
+
+type IValidatorMonitor = {
```

Comment:
we try to avoid using hungarian notation

```suggestion
type ValidatorMonitor = {
```

also might even make this an interface for more clarify

```suggestion
interface ValidatorMonitor = {
```